qiita_db/data.py

```diff
@@ -783,16 +772,15 @@ def delete(cls, ppd_id):
             if ppd.status != 'sandbox':
                 raise QiitaDBStatusError(
                     "Illegal operation on non sandboxed preprocessed data")
+            elif ppd.is_submitted_to_ebi:
```

Comment:
This case already existed before (see line 791 in the old code) and it was already tested (they're the lines that I have commented out in the test - so it is tested here: https://github.com/biocore/qiita/pull/1481/files#diff-e54a9d2a89750a099959730f348c06e1R432)